packages/server/test/migration/0.1.19-status.test.ts

```diff
@@ -0,0 +1,205 @@
+import { MigrationService, DatabaseService, ShutDownSignal } from '@botpress/messaging-engine'
+import { v4 as uuid } from 'uuid'
+import { StatusMigration } from '../../src/migrations/0.1.19-status'
+import { StatusTable } from '../../src/status/table'
+import { app, setupApp } from '../integration/utils'
+
+const MIGRATION_VERSION = '0.1.19'
+const PREVIOUS_VERSION = '0.1.18'
+
+const TABLE = 'msg_conduits'
+const COLUMN = 'initialized'
+const STATUS_TABLE = 'msg_status'
+
+const TELEGRAM_CHANNEL_ID = '0198f4f5-6100-4549-92e5-da6cc31b4ad1'
+
+describe('0.1.19 - Status', () => {
+  let migration: MigrationService
+  let database: DatabaseService
+
+  beforeAll(async () => {
+    await setupApp()
+    migration = app.migration
+    database = app.database
+
+    migration.setupMigrations([StatusMigration])
+  })
```

Comment:
Setting up the migration service with a single migration will not work in the future if another migration affects this table. Consider this scenario : in the future the status table is removed completely in a migration that creates a new table called 'state'. When running this test, the down migration for the StatusMigration wouldn't work at all because the status table would not exist. It only works here because the version of the Server code produces a Database with the expected format for the test, but it won't be the case in the future.

A solution to this would be to adjust the database schema to be what we expect by doing an initial down migration with all the migrations to the target version, and after that calling `migration.setupMigrations([StatusMigration])` to test the StatusMigration exclusively.

Reply:
Another solution would be to not tests all the migrations independently at all, and start from the lastest one and down migrate to the first one, and then back up again with some jitter in between

Reply:
Good point, we'll handle this once it becomes an issue!